spec/view_component/form/submit_component_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe ViewComponent::Form::SubmitComponent, type: :component do
+  let(:object)  { OpenStruct.new }
+  let(:form)    { form_with(object) }
+  let(:options) { {} }
+
+  let(:component) { render_inline(described_class.new(form, "", options)) }
```

Comment:
```suggestion
  let(:value) { "" }

  let(:component) { render_inline(described_class.new(form, value, options)) }
```

Test with a custom value may be nice